### DISCUSSION TITLE
Disconnected players should remain standing rather than ragdoll

### DISCUSSION THREAD

**Discussion by Anikyte** [FakeFishGames/Barotrauma#6103] [Ideas and suggestions]
Even though players stay alive and can re-enter their bodies, they often get looted since they ragdoll on disconnect. Disconnecting players should instead remain standing so that looting is not as easy and would require other players to stun/kill them, affecting their karma and trust among the crew. Further, it would prevent players standing in water from drowning before they can reconnect.

**Comment by Cogs16** [comment]
Additionally, it might be interesting to have a toggleable option to have bots take over players bodies on disconnect. Players could also take over bot bodies on join perhaps.

**Reply by Killemnocache** [reply]
._.

**Reply by Veritius** [reply]
Bots are kinda... stupid, though. It'd be very frustrating to return to a game where the bot has suffocated for... no reason. Until bots get smarter, this won't really work in practice.

**Comment by mjsonic77** [comment]
Maybe after a couple of minutes they would go limp.
